README.md

```diff
@@ -37,7 +37,8 @@ To install Mapbox Navigation using [CocoaPods](https://cocoapods.org/):
 
 1. Create a [Podfile](https://guides.cocoapods.org/syntax/podfile.html) with the following specification:
    ```ruby
-   pod 'MapboxNavigation', '~> 0.14'
+   pod 'MapboxCoreNavigation', :podspec => 'https://raw.githubusercontent.com/mapbox/mapbox-navigation-ios/v0.15.0-beta.1/MapboxCoreNavigation.podspec'
+   pod 'MapboxNavigation', :podspec => 'https://raw.githubusercontent.com/mapbox/mapbox-navigation-ios/v0.15.0-beta.1/MapboxNavigation.podspec'
```

Comment:
@bsudekum just curious, is there a reason why we're referencing podspec instead of version #? Feel like this dangerous -- say should the file get moved or folder directory gets renamed. 🤔

Reply:
@bsudekum never mind, this is for beta release.

Reply:
The URL also specifies a tag (v0.15.0-beta.1), which would only go stale if we delete the tag, which we have no plans to do.